PythonToolbox/quantconnect/__init__.py

```diff
@@ -0,0 +1,12 @@
+# QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+# Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
I don't think this file is required since it has no content apart from the added header? Each file already has it's own license banner

Reply:
I added it just because it at least used to be required for module initialization (https://docs.python.org/3/tutorial/modules.html#packages). Further reading, however, seems to imply that as of Python 3.3, an ```__init__```  would be required under the module subheading only if a script is required to be executed upon import (e.g., module depreciation warnings). For now, I'll remove that file!